_posts/2024-05-23-get-involved-with-pyos.md

```diff
@@ -0,0 +1,90 @@
+* Ability to provide a technical review of Python package structure and code quality/approach to solving the problems that the package seeks to address.
+
+We like to have a mix of technical and usability focus in our reviews so it’s ok if you don’t have all of the above skills! And if you are interested in peer review but have never reviewed before, we offer a mentorship program where we will pair you up with someone who has more experience reviewing code. From this experience you can learn more and empower yourself with code review skills. Software review skills are generally useful in data science, so they are skills worth investing in!
+
+To join our review team, [fill out our reviewer form](https://docs.google.com/forms/d/e/1FAIpQLSeVf-L_1-jYeO84OvEE8UemEoCmIiD5ddP_aO8S90vb7srADQ/viewform). We will contact you if we have a package that we need reviewers for. It’s OK if you’ve never reviewed a package before! We’ll walk you through it.
+
+### pyOpenSci's Editorial Team
+The pyOpenSci Editorial Board is comprised of a diverse group of volunteers, with each editor being responsible for the following tasks:
+
+* Finding reviewers from diverse backgrounds who have a mixture of scientific domain and Python experience.
+* Overseeing the entire review process for a package ensuring it runs in a timely and efficient manner.
+* Supporting the submitting authors and reviewers in answering questions related to the review.
+* Determining whether that package should be accepted into the pyOpenSci ecosystem once the review has wrapped up.
+
+It's critical to our mission that our Editorial Board have a combined expertise in various scientific domains, technical expertise in Python packaging, awareness of the importance of documentation in package usability, and awareness of the importance of CI/test suites in order to ensure robust software development. We do not expect any single editor to be an expert in all of these areas!
+
+### Apply to be a pyOpenSci editor
+As of May 2024, pyOpenSci looking for editors from a wide variety of scientific backgrounds due to an increased number of package submissions. If you feel you'd be a good fit for the pyOpenSci Editorial Board, please fill out [this form](https://docs.google.com/forms/d/e/1FAIpQLScRQHQ7NKVEAG3BKAphiUdVFvQ5nkez0IpyXBMZDzXjuBPloQ/viewform).
+
+If you're interested in joining our Editorial Board, but have questions or would like to chat with someone from pyOpenSci about volunteering to be an editor, please email us at [media@pyopensci.org](mailto:media@pyopensci.org).
+
+### pyOpenSci's GitHub Help Wanted board
+pyOpenSci maintains a [GitHub Help Wanted board](https://github.com/orgs/pyOpenSci/projects/3) filled with a wide variety of issues, ranging from highly technical to beginner-friendly. The status column makes it easy to identify the issue category, and the label column helps to further clarify any needs around the issue.
```

Comment:
The message here should be. more like - do you want to contribute to an open source project that is supporting other open science projects? we need your help! 

check out our help-wanted board where we .... 

all contribtors get added to our website <link to contributor paeg on website>